types/index.esm.d.ts

```diff
@@ -2823,15 +2830,15 @@ export type TimeScaleOptions = CartesianScaleOptions & {
 	 * options for creating a new adapter instance
 	 */
 	adapters: {
-		date: any;
+		date: DateAdapter;
```

Comment:
I don't think this is correct. These are the options that we can pass to the date adapter so I think it's more correct as `unknown`

Reply:
fixed